hlapipeline/alignimages.py

```diff
@@ -429,6 +433,58 @@ def update_image_wcs_info(tweakwcs_output,imagelist):
     hdulist.close()
 
 
+# ======================================================================================================================
+
+def interpret_fit_rms(tweakwcs_output, reference_catalog):
+    """Interpret the FIT information to convert RMS to physical units
+
+    Parameters
+    ----------
+    tweakwcs_output : list
+        output of tweakwcs. Contains sourcelist tables, newly computed WCS info, etc. for every chip of every valid
+        input image.
+
+    imagelist : list
+        list of valid processed images to be updated
```

Comment:
Looks like the argument documentation is incorrect.

Reply:
Fixed in latest commit.